acceptance-tests/apps/javadbapp/manifest.yml

```diff
@@ -0,0 +1,10 @@
+---
+applications:
+- name: cloudsql-java-app
+  path: apps/javadbapp/javadbapp-1.0.0.jar
+  buildpacks:
+  - java_buildpack_offline
+  env:
+    JBP_CONFIG_OPEN_JDK_JRE: '{ jre: { version: 17.+ } }'
+    JBP_CONFIG_SPRING_AUTO_RECONFIGURATION: '{ enabled: false }'
+    JAVA_OPTS: '-Djavax.net.ssl.keyStore=/app/META-INF/keystore.jks -Djavax.net.ssl.keyStorePassword=qwertyuiop'
```

Comment:
I'm guessing this isn't a security issue. Would it be possible to add a comment saying so, to avoid anyone trying to investigate the risk in the future?

Reply:
Externalised the keystore password, so it's no longer hard-coded in the application